test/tests/radio_radio.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+
```

Comment:
Hey @jongund it looks like use strict was remove here, and that is the cause of the error?